MoltenVK/MoltenVK/Commands/MVKCommandEncodingPool.mm

```diff
@@ -77,10 +77,13 @@
 	MVK_ENC_REZ_ACCESS(_cmdClearDefaultDepthStencilState, newMTLDepthStencilState(useDepth, useStencil));
 }
 
-const MVKMTLBufferAllocation* MVKCommandEncodingPool::acquireMTLBufferAllocation(NSUInteger length, bool isDedicated) {
+const MVKMTLBufferAllocation* MVKCommandEncodingPool::acquireMTLBufferAllocation(NSUInteger length, bool isPrivate, bool isDedicated) {
```

Comment:
To anticipate possible future unexpected behavior confusion, can we handle the `false, true` case (non-private dedicated), in case it ever gets attempted in the future?

For now, I'm okay with just adding an `MVKAssert()`at the top that indicates that `false, true` is not going to do what the caller is expecting.

Another option would be to combine the two flags into an enum, but that is probably overkill.